service/http.proxy/src/test/java/org/kaazing/gateway/service/http/proxy/HttpProxyLoggingWarnIT.java

```diff
@@ -55,16 +51,10 @@
     public HttpProxyLoggingWarnIT(String serviceName, String expectedMessage) {
         this.serviceName = serviceName;
         this.expectedMessage = expectedMessage;
-        this.chain = RuleChain.outerRule(new MethodExecutionTrace()).around(robot).around(checkLogMessageRule).around(getGatewayRule());
+        this.chain = RuleChain.outerRule(new MethodExecutionTrace()).around(getGatewayRule()).around(checkLogMessageRule).around(robot);
```

Comment:
Yes. We need to let the gateway process all events. If we have the checkLogMessageRule  around the gateway, the gateway will be stopped before doing the check. No matter how long you wait, nothing will change as the gateway is already stopped.